code/game/objects/items/weapons/flamethrower.dm

```diff
@@ -26,6 +26,7 @@
 	var/create_full = FALSE
 	var/create_with_tank = FALSE
 	var/igniter_type = /obj/item/device/assembly/igniter
+	var/trigger_guard = TRIGGER_GUARD_NORMAL
```

Comment:
please do not define variables on different types like this, define it once on their shared parent.